docbook/Admin_Guide/en-US/config/date.xml

```diff
@@ -43,31 +43,6 @@
 				</para>
 			</listitem>
 		</varlistentry>
-		<varlistentry>
-			<term>$g_datetime_picker_format</term>
```

Comment:
You should update doc for *$g_normal_date_format*, add a note similar to https://github.com/mantisbt/mantisbt/pull/1587/files#diff-dccc8d20b0765881557a770f05f6acb0R1308 explaining that it also covers the datetime picker.

Reply:
Done

Reply:
Thanks